docs/src/conceptual/solution-selection-catalogs.rst

```diff
@@ -0,0 +1,189 @@
+      ├── TensileLibrary_lazy_gfx1030.yaml                   # [A]
+      ├── TensileLibrary_lazy_gfx900.yaml                                    
+      ├── TensileLibrary_lazy_gfx906.yaml                                    
+      ├...
+      ├── TensileLibrary_Type_..._fallback_gfx1030.hsaco
+      ├── TensileLibrary_Type_..._fallback_gfx900.hsaco
+      ├── TensileLibrary_Type_..._fallback_gfx906.hsaco
+      ├── TensileLibrary_Type_..._fallback.yaml              # [B]
+      ├── TensileLibrary_Type_..._gfx900.co
+      ├── TensileLibrary_Type_..._gfx900.hsaco
+      ├── TensileLibrary_Type_..._gfx900.yaml                # [C]
+      ├── TensileLibrary_Type_..._gfx906.co
+      ├── TensileLibrary_Type_..._gfx906.yaml                # [D]
+
+Line **[A]** shows the parent catalog for gfx1030, the first of the three parent catalogs generated.
+Line **[B]** shows a fallback child catalog, which reference each of the archiecture specific fallback kernels
+in the associated .hsaco files.
+This means that at least some of the parameter/problem type combinations for *DD_Contraction_l_Alik_Bjlk_Cijk_Dijk*
+haven't been explicitly tuned for these architectures.
+Note that the matching .hsaco files (above **[B]**) are code object libraries for HIP source kernels.
+These files are referenced by the fallback catalog.
+Line **[C]** shows a child catalog for gfx900 that references both HIP source and assembly source kernels, found in the associated .hsaco and .co files, respectively.
+Line **[D]** shows a child catalog for gfx906, similar to the gfx900 catalog. However, notice that there is only one associated
+.co file. This means that there are only assembly source kernels in this catalog.
+
+**Example: Parent solution selection catalog**
+
+.. code-block:: yaml
+  :caption: build/library/TensileLibrary_lazy_gfx900.yaml
+
+  library:
+    rows:                                                    # [A_]
+    - library:
+        map:
+          Contraction_l_Alik_Bjlk_Cijk_Dijk:                 # [B_]
+            ...
+            rows:                                            # [C_]
+            - library: {type: Placeholder, value: TensileLibrary_Type_SS_..._fallback}
+              predicate:
+                type: And
+                value:
+                - type: TypesEqual
+                  value: [Float, Float, Float, Float]
+                - {type: HighPrecisionAccumulate, value: false}
+                - {type: F32XdlMathOp, value: Float}
+                - {type: StochasticRounding, value: false}
+            - ...
+            type: Problem
+            ...
+          Contraction_l_Alik_Bljk_Cijk_Dijk:
+            rows:
+              - ...
+            type: Problem                                    # [_C]
+        property: {type: OperationIdentifier}
+        type: ProblemMap                                     # [_B]
+      predicate: {type: TruePred}
+    type: Hardware                                           # [_A]
```

Comment:
@bstefanuk Why are [_C], [_B], [_A] needed? Remove them if not referenced.

Reply:
Visual helper to show opening and closing of sections. There could be a better way to do this. They are referenced below.